main.js

```diff
@@ -0,0 +1,42 @@
+// global variable for outfit boxes
+var nakedBear = document.querySelector('.main-bear');
+var hats = document.querySelector('.hats-selection');
+var clothes = document.querySelector('.clothes-selection');
+var accessories = document.querySelector('.accessories-selection');
+var background = document.querySelector('.background-selection');
+
+// event listener on options selection
+hats.addEventListener('click', addHats);
+clothes.addEventListener('click', addClothes);
+accessories.addEventListener('click', addAccessories);
+background.addEventListener('click', addBackground);
+
+// add hats
+function addHats(e){
+ if(e.target.classList.contains('hat-Options')){
+
+   console.log(1);
```

Comment:
Good addition to the Javascript.  This will allow us to condense the coding for all of the button features in the project.  This will help reduce the overall amount javascript code at the end of the project.  Is there any way to reduce the overall event listeners amount doing this method?